CHANGELOG.md

```diff
@@ -1,21 +1,6 @@
-# v18
-
-**New Feature:**
-- Support leader election. [kubernetes-client/python-base#206](https://github.com/kubernetes-client/python-base/pull/206)
-
-**Bug Fix:**
-- Raise exception when an empty config file is passed to load_kube_config. [kubernetes-client/python-base#223](https://github.com/kubernetes-client/python-base/pull/223)
-- fix: load cache error when CacheDecoder object is not callable. [kubernetes-client/python-base#226](https://github.com/kubernetes-client/python-base/pull/226)
-- Fix Watch retries with 410 errors. [kubernetes-client/python-base#227](https://github.com/kubernetes-client/python-base/pull/227)
-- Automatically handles chunked or non-chunked responses. Fix ResponseNotChunked error from watch. [kubernetes-client/python-base#231](https://github.com/kubernetes-client/python-base/pull/231)
-
-**API Change:**
-- Add allowWatchBookmarks, resoureVersionMatch parameters to custom objects. [kubernetes-client/gen#180](https://github.com/kubernetes-client/gen/pull/180)
+# v18.17.0a1
 
-
-# v18.0.0-snapshot
```

Comment:
so there is no snapshot? i noticed 1.17 below still stay at snapshot

Reply:
Cherry-picked https://github.com/kubernetes-client/python/pull/1322 (just merged into master today) to rename the 1.17 snapshot. 

There should be no "snapshot" section in the CHANGELOG.md in a release branch. A "snapshot" section is a placeholder for a WIP release, it can only exists in the master branch, and should be renamed when we cut a release. 

The correct order should have been:
- master branch has 17.0.0-snapshot
- release-17.0 branch renames 17.0.0-snapshot into v17.14.0a1
- https://github.com/kubernetes-client/python/pull/1322 cherrypicks the rename back to master branch
- master branch has 18.0.0-snapshot (**this didn't happen, we did everything in release-18.0 and left the master branch behind**)
- release-18.0 branch renames 18.0.0-snapshot into v18.17.0a1 (**we are here**)
- cherrypick the rename back to master branch (**this is not possible yet. Master branch is still at 17.0.0-snapshot. I need to make the master branch catch up**)

Reply:
ok. so the problem is there is only one 'master' branch, which would force the sequential ordering of the releases, that makes parallel releases painful. i think we either fix the above process to make parallel releases easy, or we do just one release a time (once we catch up with the k/k releases).

Reply:
the sole purpose of this PR is to have the correct release note for v18.17.0a1 in https://github.com/kubernetes-client/python/blob/release-18.0/CHANGELOG.md, which is the linked place we documented in https://github.com/kubernetes-client/python/releases/tag/v18.17.0a1

Reply:
> i think we either fix the above process to make parallel releases easy, or we do just one release a time (once we catch up with the k/k releases)

My plan is to do the latter using scripts which automate the release, once we clean up the parallel 17 and 18 release. Everything starting v18.x.0b1 can be sequential